src/pages/seyeonPark/Login/Login.js

```diff
@@ -1,8 +1,53 @@
-import React, { Component } from 'react';
+import React from 'react';
+import LoginBox from './LoginComponent/LoginBox';
+import '../../../pages/seyeonPark/Login/Login.scss';
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      inputIdValue: '',
+      inputPwValue: '',
+      isButtonOn: false,
+    };
+  }
+  // 이벤트 핸들러가 제대로 불리는지 콘솔로 꼭 찍어보기
+  handleInput = event => {
+    const { name, value } = event.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  changeButton = () => {
+    this.state.inputIdValue.indexOf('@') !== -1 &&
+    this.state.inputPwValue.length >= 5
+      ? this.setState({ isButtonOn: true })
+      : this.setState({ isButtonOn: false });
+  };
+
+  goToMain = () => {
+    this.props.history.push('/main-seyeon');
+  };
+
   render() {
-    return null;
+    const { isButtonOn } = this.state;
+    return (
+      <div className="loginContainer">
+        <main className="container">
+          <h1>Westagram</h1>
+          <LoginBox onChange={this.handleInput} changeBtn={this.changeButton} />
+          <button
+            className={isButtonOn ? 'changeBtnColor' : 'button'}
+            onClick={this.goToMain}
+            disabled={isButtonOn ? false : true}
```

Comment:
```suggestion
            disabled={!isButtonOn}
```
이렇게 표현할 수 있습니다 Boolean 값의 특성을 잘 이용해보세요!

Reply:
몰랐던 부분인데 Boolean 값의 특성을 다시 한번 공부하도록 하겠습니다.